.evergreen/socks5srv.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
```

Comment:
Also leaving a comment for context, the PR to add this to drivers-evergreen-tools is in https://github.com/mongodb-labs/drivers-evergreen-tools/pull/181